src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+import AddTodo from './components/AddTodo'
+import Hints from './components/Hints'
+import Title from './components/Title'
+import TodoInput from './components/TodoInput'
+import TodosList from './components/TodosList'
+import Wrapper from './components/Wrapper'
+
+function App() {
+  const [showInput, setShowInput] = useState(false)
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (title) =>{
+    setTodos(v => [...v, {id: (v.at(-1)?.id ?? 0) + 1, title, done: false}])
```

Comment:
I think it should be a good idea to use meaningful names, could you use a more descriptive name for this argument "v"?